sources/packages/backend/apps/queue-consumers/src/services/application/application.service.ts

```diff
@@ -153,4 +153,16 @@ export class ApplicationService {
         .getMany()
     );
   }
+
+  async getEligibleApplicationsForNotification(): Promise<Application[]> {
```

Comment:
If the PR is related to only scheduler creation, please remove these files from the PR.

Reply:
If the file is present in the PR it must be good enough to be merged to main. No temporary or in-progress code should be present unless it is a partial implementation of the final approach, wich does not seem to be the case here.